src/NodeJS/Server.php

```diff
@@ -254,6 +262,52 @@ public function getConnection()
     }
 
     /**
+     * Return the options.
+     *
+     * @return array
+     */
+    public function getOptions()
```

Comment:
Looking at order of existing setters/getters shows that all getters are grouped into one group, and all setters are grouped into another group. Could you please move your setter/getter to comply with that?